src/value/link.ts

```diff
@@ -102,10 +102,9 @@ export const link = <
 						(model.config?.allowTargetBlank && faker.boolean())
 							? "_blank"
 							: undefined,
-					text:
-						config.withText ?? (model.config?.allowText && faker.boolean())
-							? sentenceCase(faker.words(2))
-							: undefined,
+					...(config.withText ?? (model.config?.allowText && faker.boolean())
```

Comment:
this was wrong, as soon as i upgraded prismic mock in prismic client, some tests failed because of this

iirc, in other places we don't send the text property at all if the model doesn't allow it